src/app/pages/questions/components/question/range-info-input/range-info-input.component.ts

```diff
@@ -0,0 +1,50 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+
+import { Item, Response } from '../../../../../shared/models/question'
+
+let uniqueID = 0
+
+@Component({
+  selector: 'range-info-input',
+  templateUrl: 'range-info-input.component.html'
+})
+export class RangeInfoInputComponent implements OnInit {
```

Comment:
Most of this class has duplicate code of the RangeInputComponent. Can we maybe extend the class ?

Reply:
Yup. Made the changes.